recipes/skforecast/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "skforecast" %}
+{% set version = "0.14.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/skforecast-{{ version }}.tar.gz
+  sha256: c96147f0d353f2fe07dd7ba2b9561b03c426fe6b8000d60f0f3728d53322e9eb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=61
+    - toml
+    - python-build
+    - pip
+  run:
+    - python >=3.9
+    - numpy >=1.22
+    - pandas >=1.5
+    - tqdm >=4.57
+    - scikit-learn >=1.2
+    - optuna >=2.10
+    - joblib >=1.1
+    - numba >=0.59
+
+test:
+  imports:
+    - skforecast
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https:skforecast.org
```

The URL isn't formatted correctly.
```suggestion
  home: https://skforecast.org
```